src/services/nnas/routes/people.ts

```diff
@@ -64,6 +65,33 @@ router.post('/', ratelimit, deviceCertificateMiddleware, async (request: express
 
 	const person: Person = request.body.person;
 
+	// TODO - This is kinda ugly
+	const birthdate = new Date(person.birth_date);
+	const today = new Date();
+	const eighteenthBirthday = new Date(birthdate);
+	eighteenthBirthday.setFullYear(birthdate.getFullYear() + 18);

```

Comment:
We should probably move this into a util function - I don't think it's ugly like the comment says - we can just put logic somewhere so we don't have to do it every time

Maybe a `getAgeFromDate(date)` function - which we can use smth like this:

```js
const age = getAgeFromDate(date);
if (age < 18) {
  // do things
}
```

Would allow for reuse with other age verification (like < 16, < 13)

---

Speaking of which, should we add a < 13 block to this code?

Reply:
I agree with the util function. I think that will be useful for the other age checks like you said, but right now I'd like to keep this PR focussed on the Mississippi changes if possible

Reply:
So to clarify:
- You'll make the util function and implement it in this PR
- For the < 13 check, that will be for a different PR

Is that right?

Reply:
Correct 👍

Reply:
To clarify, I want to do separate PRs for different laws/regions because they all vary in certain ways, and given how important the features are I want the PRs to be as focussed as possible on each law to make sure everything is nailed down correctly